clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/description/manifest/KubernetesManifestAnnotater.java

```diff
@@ -208,12 +208,13 @@ public static KubernetesManifestTraffic getTraffic(KubernetesManifest manifest)
         .build();
   }
 
-  public static KubernetesCachingProperties getCachingProperties(KubernetesManifest manifest) {
+  public static KubernetesCachingProperties getCachingProperties(KubernetesManifest manifest, boolean onlySpinnakerManaged) {
     Map<String, String> annotations = manifest.getAnnotations();
-
+    boolean shouldIgnore = getAnnotation(annotations, IGNORE_CACHING, new TypeReference<Boolean>() {}, false) ||
+      (onlySpinnakerManaged  && getAnnotation(annotations, APPLICATION, new TypeReference<String>() {}, "").equals(""));
```

Comment:
Extra space before `&&`.

I don't think this belongs in the manifest annotator, I think we should read the properties set from the annotations here, and then make a decision about whether or not to cache this outside the annotator.

Reply:
That makes sense. It did feel like a bit of a hack. I will look around and make sense of how that is decided.

I now realize that submitting a pull request from my master branch is not ideal. I would like to submit a new one once I fix the implementation.

Reply:
Sounds good